test/test_rosbag/package.xml

```diff
@@ -7,36 +11,39 @@
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>BSD</license>
 
-  <url>http://ros.org/wiki/rosbag</url>
+  <url type="website">http://wiki.ros.org/rosbag</url>
+  <url type="bugtracker">https://github.com/ros/ros_comm/issues</url>
+  <url type="repository">https://github.com/ros/ros_comm</url>
   <author>Tim Field</author>
   <author>Jeremy Leibs</author>
   <author>James Bowman</author>
 
+  <depend>rosbag</depend>
+  <depend>rostest</depend>
+
   <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
 
   <build_depend>boost</build_depend>
   <build_depend>bzip2</build_depend>
   <build_depend>cpp_common</build_depend>
   <build_depend>message_generation</build_depend>
-  <build_depend>python-imaging</build_depend>
-  <build_depend>rosbag</build_depend>
+  <build_depend condition="$ROS_PYTHON_VERSION == 2">python-imaging</build_depend>
+  <build_depend condition="$ROS_PYTHON_VERSION == 3">python3-imaging</build_depend>
```

Comment:
I think the equivalent python3 key is [`python3-pil`](https://github.com/ros/rosdistro/blob/0ea30fed0bcb564b9e4d5b2d2efd8783e8e3afa9/rosdep/python.yaml#L5107). There's no `python3-imaging` key (nor is there a debian package with that name).

Reply:
Good catch: 85394ffe89f981469bbd0954d1483382aff20df8.